src/workflows/withdrawal/completeERC721Withdrawal.ts

```diff
@@ -263,7 +255,7 @@ export async function completeERC721WithdrawalWorkflow(
       ),
     )
     .catch(error => {
-      if (error.response.status === 404) {
+      if (error.response?.status === 404) {
         console.log(error.response); // token is already minted on L1
```

Comment:
Do we want this log?

Reply:
[1df7ea9](https://github.com/immutable/imx-core-sdk/pull/185/commits/1df7ea9f71888977c0a6ffa4a1b00e6851a63039)